libs/community/langchain_community/document_loaders/github.py

```diff
@@ -65,7 +66,7 @@ class GitHubIssuesLoader(BaseGitHubLoader):
     """Only show notifications updated after the given time.
         This is a timestamp in ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ."""
 
-    @validator("since")
+    @validator("since", allow_reuse=True)
```

Comment:
otherwise gives error in PR jobs: 
```
ERROR tests/unit_tests/document_loaders/loaders/vendors/test_docugami.py - pydantic.errors.ConfigError: duplicate validator function "langchain_community.document_loaders.github.GitHubIssuesLoader.validate_since"; if this is intended, set `allow_reuse=True`
[1058](https://github.com/langchain-ai/langchain/actions/runs/7354885850/job/20022700200?pr=15305#step:5:1059)
```